test/evergreen/evg_cfg.py

```diff
@@ -264,7 +264,7 @@ def evg_cfg(action, test_type):
     """
 
     # Make sure the program is run under a checkout of wiredtiger repository
-    if run('git config remote.origin.url') != WIREDTIGER_REPO:
+    if run('git config remote.origin.url') in WIREDTIGER_REPO:
```

Comment:
When I run the git command I get a string different from the one that's expected by the script (because I connect via ssh):
```
% git config remote.origin.url
ssh://git@github.com/wiredtiger/wiredtiger.git
```
I was trying to change it so both mine and yours would be accepted.